test/README.md

```diff
@@ -0,0 +1,7 @@
+# Running tests
+
+1. First you need to be authenticated into a cluster (with cluster-admin permissions)
```

Comment:
Out of curiousity, why does it need cluster-admin?

Reply:
Shipshape creates namespaces in the cluster and (eventually) some cluster level resources, like the CRDs.

Reply:
Of course, makes sense!